api/src/lib.rs

```diff
@@ -7,3 +7,6 @@ pub use metrics::Metrics;
 pub use resume_token::ResumeToken;
 
 pub use server::{AccessInterestStore, AccessModelStore, Server};
+
+#[cfg(test)]
+mod test;
```

Comment:
nit, its common to call this module `tests` as it contains many tests.

Reply:
Appreciate the nits